### PR TITLE
Create context

### PR DESCRIPTION
**Pull request by nafistiham**

### CODE REVIEW

File: src/content/reference/react/createContext.md [hunk 1]
```diff
@@ -16,38 +16,38 @@ const SomeContext = createContext(defaultValue)
...
-* `defaultValue`: The value that you want the context to have when there is no matching context provider in the tree above the component that reads context. If you don't have any meaningful default value, specify `null`. The default value is meant as a "last resort" fallback. It is static and never changes over time.
+* `defaultValue`: এটা হল সেই ভ্যালু যেটা আপনি চান যে কনটেক্সটের থাকুক। কখন? যখন সেই কনটেক্সট যেই component read করবে, ট্রিতে তার উপরে কোন ম্যাচিং কনটেক্সট প্রোভাইডার নেই। যদি আপনার কোন অর্থবহ default ভ্যালু না থাকে, ভ্যালুটা `null` করে দেন। ডিফল্ট ভ্যালু একদম "last resort" ফলব্যাক হিসেবে রাখা হয়। এটা স্ট্যাটিক এবং সময়ের সাথে অপরিবর্তিত থাকে।
 
-#### Returns {/*returns*/}
+#### রিটার্ন {/*returns*/}
 
-`createContext` returns a context object.
+`createContext` একটি কনটেক্সট অবজেক্ট রিটার্ন করে। 
 
-**The context object itself does not hold any information.** It represents _which_ context other components read or provide. Typically, you will use [`SomeContext.Provider`](#provider) in components above to specify the context value, and call [`useContext(SomeContext)`](/reference/react/useContext) in components below to read it. The context object has a few properties:
```
Comment (nomandhoni-cs) [inline src/content/reference/react/createContext.md:41]: @nafistiham  Bhai,, 
Translation is 100% okay, Apni just ei 41th line ta miss kore gesen.

**Review by nomandhoni-cs — COMMENTED** [review]
@nafistiham  Bhai,,
Translation is 100% okay, Apni just ei 41th line ta miss kore gesen.

File: src/content/reference/react/createContext.md (hunk 1 above)
Comment (nafistiham) [reply src/content/reference/react/createContext.md:41]: Done vai. Thanks a lot for pointing it out. These days, I rarely have the time to finish a page in one sitting. So, missing lines in between. Haha.

Comment (nafistiham) [reply src/content/reference/react/createContext.md:41]: @nomandhoni-cs vai

**Review by nomandhoni-cs — APPROVED** [review]
@nafistiham Bhai, Thanks a lot....

Amar review korte ektu late hoise exam coltese tai..

**Pull request merged**